rules/windows/persistence_remote_password_reset.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/10/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an attempt to reset an account password remotely. Adversaries may manipulate account passwords to maintain
+access or evade password duration policies and preserve compromised credentials.
+"""
+false_positives = ["Legitimate remote account administration."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
```

Comment:
would system integration work as well for this?

Reply:
Yes